scarb/src/bin/scarb/args.rs

```diff
@@ -330,6 +330,10 @@ pub struct PackageArgs {
     #[arg(long)]
     pub allow_dirty: bool,
 
+    /// Don't verify the contents by building them
```

Comment:
```suggestion
    /// Do not verify the contents by building them.
```

Reply:
Actually this is a help info, so no period at the end.

Reply:
Clap removes trailing periods :) So that you get proper documentation comments & help info at once. Take a look at other comments

Reply:
Okay, nervermind then